line_profiler.py

```diff
@@ -369,7 +369,11 @@ def magic_lprun(self, parameter_s=''):
 def load_ipython_extension(ip):
     """ API for IPython to recognize this module as an IPython extension.
     """
-    ip.define_magic('lprun', magic_lprun)
+    if hasattr(ip, 'define_magic'):
```

Comment:
Hmmm, are you sure you're on my branch (and reloaded the modules)?

The reason I'm asking is that is exactly the reason why I made this pull request.